src/components/Footer.jsx

```diff
@@ -98,29 +88,23 @@ const Footer = () => {
     <StyledFooter>
       <div className="footer-content">
         <div className="footer-links">
-          <div className="donate-button-container">
-            <a
-              href="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site"
-              target="blank"
-              rel="noopener noreferrer"
-              className="button donateButton label-3"
-            >
-              Donate
-            </a>
-
-            <img src={longArrowAltRight} alt="" />
-          </div>
+          <Button
```

Comment:
I may need to debug this button.  The wrong arrow appears here and on hover it blends into the background.